.changeset/little-ants-sort.md

```diff
@@ -0,0 +1,5 @@
+---
+'react-select': patch
+---
+
+Fixed 3 unit test test cases instead of skipping them
```

Comment:
Maybe we don't need a changeset for this one because it's not modifying anything that a user of `react-select` would use?

Reply:
I wasn't sure since it lives in the source, but also doesn't contribute or change any functionality so id be in favor of removing it.